### ISSUE TITLE
AI Pilot doesn`t work with Afterburners engines

### ISSUE THREAD

**Issue by D4rd0** [PapaJoesSoup/BDArmory#285]
##### ISSUE TYPE

 - Support  ( Need help getting something to work)
 - Bug Report   (Something is broken)

##### KSP and BDA Version

KSP 1.3

BDAc 0.3.0.0

##### OS / ENVIRONMENT

Windows 10

##### SUMMARY

When you activate AI pilot in a plane with the Afterburners J-404 Panther Engines (the most used engines for jets) the AI pilot throttles all the way down and does not take off. This only happens with the afterburners engines, with others engines it works fine. 
If you activate AI during flight, it throttles all the way down and lands somewhere.
Standby mode is off and activated engines manually btw. 
I have reinstalled the game and the mod several times but same happens.
If this info helps, I tested that in ksp 1.2.2 (with the latest BDAc for that version) and AI pilot works fine with afterburners.
Sorry about my english.

##### STEPS TO REPRODUCE

Just use a plane with afterburners J-404 Panther and activate AI pilot.

##### Log Files

[output_log.txt](https://github.com/PapaJoesSoup/BDArmory/files/1226779/output_log.txt)

GitHub doesn`t support .log files damn, I zipped it:
[KSP.zip](https://github.com/PapaJoesSoup/BDArmory/files/1226782/KSP.zip)

**Comment by DoctorDavinci** [comment]
You have BDAc v2.1.2 installed and NSI's weapons pack ... from what I understand, NSI's weapons pack has a .dll and it is compiled against BDAc v0.3.0.0 which could be part of the issue

Please remove NSI's weapon extention as well as any other mods not maintained by the BDAc team and update BDAc to the latest version (BDAc v0.3.0.0) .... then test to confirm your findings

I have no problems with both BDAc v0.2.1.2 and v0.3.0.0 AI Pilot's being able to use the afterburners on a craft

**Comment by NextStarIndustries** [comment]
I agree with @DoctorDavinci remove everything and just install BDAc and retest. I also have no problems with either 0.2.1.2 or 0.3.0.0 using afterburners, as they are on all of my vessel builds.

**Comment by D4rd0** [comment]
Thanks for the answers. Finally I solved it, the problem was that i was using the spanish version of the game. I checked the names of the engines that i was using and casually the only name who changes from english to spanish version is the afterburners engine, so I think that is the focus of the problem. Thanks a lot anyways.

**Comment by jrodrigv** [comment]
@D4rd0 I think we don't currently support any language different that English. There might be other issues.

I remember I did the fix for that issue.

From a coding point of view, there is a method checking this:
` private static bool IsAfterBurnerEngine(MultiModeEngine engine)
        {
            if (engine == null)
            {
                return false;
            }
            if (!engine)
            {
                return false;
            }      
            return engine.primaryEngineID == "Dry" && engine.secondaryEngineID == "Wet";
        }`

[Spanish] Quizas es posible que en la version en Español hayan cambiado las palabras por "Seco" y "Mojado" ?

**Comment by gomker** [comment]
Re-opening , still reports of issue with localization, changing title.
